src/js/view/captionsrenderer.js

```diff
@@ -352,7 +352,7 @@ const CaptionsRenderer = function (viewModel) {
     function loadWebVttPolyfill() {
         return require.ensure(['polyfills/webvtt'], function (require) {
             _WebVTT = require('polyfills/webvtt').default;
-        }, chunkLoadErrorHandler(121), 'polyfills.webvtt');
```

Comment:
@johnBartos @robwalch 2 things to address here.
1. In our Error List/Jira ticket we have set the code 301121 for vttparser.js and 301131 for polyfills.webvtt.js - considering this code used to be "121" this must have been a mistake and we should just update our documentation.
2. Need to fix the code here from 300 to 301

Reply:
disregard - closing this out and making the typo fix